content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md

```diff
@@ -0,0 +1,88 @@
+
+On top of that, we put in a great deal of effort into [improving resource support](https://github.com/kubernetes/dashboard/issues/5232). Meanwhile, we do offer support for most of the Kubernetes resources. Also, the Kubernetes Dashboard supports multiple languages: English, German, French, Japanese, Korean, Chinese (Traditional, Simplified, Traditional Hong Kong). Persian and Russian language translations are currently in work. Moreover, we are working on the support for 3rd party themes and the design of the app in general. As you can see, quite a lot of things are going on.
+
+Luckily, we do have regular contributors with domain knowledge who are taking care of the project, updating the Helm charts, translations, Go modules, and more. But as always, there could be many more hands on deck. So if you are thinking about contributing to Kubernetes, keep us in mind ;)
+
+## What’s Next
+
+The Kubernetes Dashboard has now been growing and prospering for more than 5 years now. It provides the community with an intuitive Web UI, thereby decreasing the complexity of Kubernetes and increasing its accessibility to new community members. We are proud of what the project has achieved so far, but this is by far not the end. These are our priorities for the future:
+
+*   Keep providing support for the new Kubernetes versions
+*   Keep improving the support for the existing resources
+*   Keep working on auth system improvements
+*   [Rewrite the API to use gRPC and shared informers](https://github.com/kubernetes/dashboard/pull/5449): This will allow us to improve the performance of the application but, most importantly, to support live updates coming from the Kubernetes project. It is one of the most requested features from the community.
+*   Split the application into two containers, one with the UI and the second with the API running inside.
+
+## The Kubernetes Dashboard in Numbers
+
+*   Initial commit made on October 16, 2015
+*   Over 100 million pulls from Dockerhub since the v2 release
+*   8 supported languages and the next 2 in progress
+*   Over 3360 closed PRs
+*   Over 2260 closed issues
+*   100% coverage of the supported core Kubernetes resources
+*   Over 9000 stars on GitHub
+*   Over 237 000 lines of code
+
+## Join Us
+
+As mentioned earlier, we are currently looking for more people to help us further develop and grow the project. We are open to contributions in multiple areas, i.e., [issues with help wanted label](https://github.com/kubernetes/dashboard/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22). Please feel free to reach out via GitHub or the #sig-ui channel in the Kubernetes Slack.
+
+Learn more about [Kubermatic’s upstream contributions and open source projects](https://www.kubermatic.com/company/community/).
```

Comment:
With the [CNCF website guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md) in mind, I strongly recommend revising or removing this paragraph and hyperlink.

> It is OK to say that, e.g., “Prometheus was originally created by Soundcloud” or “Kubernetes builds upon 15 years of experience of running production workloads at Google,” but the origin company should not otherwise be referred to on the project homepage.

That's talking about home pages, but I'd like to see the same intent captured in how we publish articles on the blog. 
So, for example, “The Kubernetes Dashboard was originally created by Google” would be fine, but this link to Kubermatic feels like it wouldn't be.


See the [Kubernetes content guide](https://kubernetes.io/docs/contribute/style/content-guide/) for more on this.

Reply:
Sure, that's totally understandable. We weren't sure if this would be accepted anyway. I will simply remove that part.